partcad-cli/src/partcad_cli/click/command.py

```
@@ -160,7 +161,7 @@ def cli(ctx, v, q, no_ansi, p, format):
 
 
 @cli.result_callback()
-def process_result(result, v, q, no_ansi, p, format):
+def process_result(result, verbose, quiet, no_ansi, package, format):
```

Ruff (0.8.2)

Missing return type annotation for public function `process_result`

Add return type annotation: `None`

(ANN201)